.github/workflows/ci.yaml

```
@@ -8,7 +8,7 @@ on:
   pull_request: ~
 
 env:
-  CACHE_VERSION: 4
+  CACHE_VERSION: 4  # Also change this in the primer workflow
```

```suggestion
  # Also change CACHE_VERSION in the primer workflow
  CACHE_VERSION: 4
```
`prettier` doesn't seem to like two spaces 😞

---

Oh wel.. 😢